src/main/proto/ga4gh/bio_metadata.proto

```diff
@@ -155,19 +161,6 @@ message BioCharacteristic {
   //      termId: "HP:0001714"
   //      label: "Ventricular hypertrophy"
   //
-  repeated OntologyTerm negatedOntologyTerms = 3;
-
-}
-
-// Characteristics is a wrapper object for the diseases, phenotypes associated
-// with a first class BioMetadata object (Individual, BioSample).
-message Characteristics {
-  // Diseases associated with the parent BioMetadata object (Individual,
-  // BioSample).
-  repeated BioCharacteristic diseases = 1;
-
-  // Phenotypes associated with the parent BioMetadata object (Individual,
-  // BioSample).
-  repeated BioCharacteristic phenotypes = 2;
+  repeated OntologyTerm negatedOntologyTerms = 4;
 
```

Comment:
Small comment on 'characteristics_class': I would avoid 'class' and use maybe 'scope' to match the documentation. I would also suggest to replace 'typical examples could be' with a stronger requirement, such as a list of acceptable values (maybe with a definition, for example in the above I'm not sure of the difference between a phenotype and an observation - does it matter?)